source/nodejs/ac-typed-schema/src/markdown/generate-markdown.ts

```diff
@@ -21,7 +52,7 @@ export function createPropertiesSummary(classDefinition: SchemaClass, knownTypes
 		properties = sortProperties(properties);
 
 		if (includeVersion && defined(elementVersion) && elementVersion != "1.0") {
-			md += "**Introduced in version " + elementVersion + "**\n\n";
+			md += __("**Introduced in version %s**\n\n", elementVersion);
```

Comment:
In the future, this hardcoded string (and all others) will be replaced by localized resources right? But that is out-of-scope for this PR

Reply:
the way this library works is that you use a string literal that's the key for the translation backend. effectively, you write your hardcoded strings in your native language and use those as the default translations. under the hood, the library will use the locale-appropriate string as specified.

Reply:
Cool, thanks for clarifying!